types/parse/v1/parse-tests.ts

```diff
@@ -148,7 +148,7 @@ function test_query() {
     const testQuery = Parse.Query.or(query, query);
 }
 
-class TestCollection extends Parse.Collection<Object> {
+class TestCollection extends Parse.Collection<Parse.Object> {
```

Comment:
Looking at where this is being used a bit more and trying this potential change out, it looks like this generic could also be updated to more specifically reference the `Game` test class instead if that would be preferable. If I were to do that, I'd probably also update the constructor to take `Game`s as well.